be/src/Unic.UrlMapper2/code/Services/Sanitizer.cs

```diff
@@ -7,7 +7,8 @@ public class Sanitizer : ISanitizer
     {
         public virtual void SanitizeRedirectSearchData(RedirectSearchData redirectSearchData)
         {
-            redirectSearchData.SourceTerm = this.SanitizeTerm(redirectSearchData.SourceTerm);
+            redirectSearchData.SourceTerm = this.SanitizeTerm(redirectSearchData.SourceTerm)?.ToLower();
```

Comment:
**issue:** The `ToLower()` method call was considered to be part of the sanitation, but now is hiding a bit outside of that method.
**suggestion:** Make the `SanitizeTerm` method accept an additional parameter (e.g. `convertToLower`) and give it a default value of `true` to not break backwards compatibility.

Reply:
done